google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -726,13 +726,11 @@ public ReadChannel reader(BlobSourceOption... options) {
    * <pre>{@code
    * byte[] content = "Hello, World!".getBytes(UTF_8);
    * try (WriteChannel writer = blob.writer()) {
-   *   try {
    *     writer.write(ByteBuffer.wrap(content, 0, content.length));
-   *   } catch (Exception ex) {
-   *     // handle exception
-   *   }
+   * } catch (IOException ex) {
+   *   // handle exception
    * }
-   * }</pre>
+   * </pre>
```

Comment:
You do need this `}</pre>` closing bracket to close `{@code... }` on line 726

@elharo 
![dpyxdV2b95J](https://user-images.githubusercontent.com/1907648/81841771-cae81480-94ff-11ea-9194-a78dde855e56.png)

Reply:
Done